app/services/efile/poll_for_acknowledgments_service.rb

```diff
@@ -1,10 +1,27 @@
 module Efile
   class PollForAcknowledgmentsService
     def self.run
-      transmitted_submissions = EfileSubmission.in_state(:transmitted).limit(100)
-      response = Efile::GyrEfilerService.run_efiler_command(Rails.application.config.efile_environment, "acks", *transmitted_submissions.pluck(:irs_submission_id))
+      ack_count = 0
+      submission_ids = transmitted_submission_ids
+      submission_ids.each_slice(100) do |submission_ids|
+        ack_count = _handle_response(Efile::GyrEfilerService.run_efiler_command(Rails.application.config.efile_environment, "acks", *submission_ids))
```

Comment:
does this handle the error we were seeing when there are no submissions? I think it should, (the loop wont run if there aren't any items in the first slice, i assume?) but just wanted to make sure.